File: Hishab Server/hishabApp/hishabApp/src/main/java/com/example/hishab/hishabApp/apiModels/ShoppingResponse.java
package com.example.hishab.hishabApp.apiModels;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class ShoppingResponse {
	private int shoppingId;
	private int localId;
	private List<ShoppingItemResponse>itemResponses;
	
}

File: Hishab/app/src/main/java/com/example/hishab/retrofit/response/shoppingResponse.kt
package com.example.hishab.retrofit.response


import com.example.hishab.retrofit.response.ShoppingItemResponse

class shoppingResponse {
    var shoppingId = 0
    var localId = 0L
    var itemResponses: List<ShoppingItemResponse>? = null

    constructor() {}
    constructor(shoppingId: Int, itemResponses: List<ShoppingItemResponse>?) {
        this.shoppingId = shoppingId
        this.itemResponses = itemResponses
    }
}